### PR TITLE
[18.0][MIG] stock_packaging_calculator: Migration to 18.0

### PR DESCRIPTION
**Pull request by thienvh332**
### Ported PRs
- **#1407**
- **#1902**

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo_test_helper
```
Comment (rousseldenis) [inline test-requirements.txt:1]: @thienvh332 This one you can add it with a normal commit. Thanks

**Review by rousseldenis — APPROVED** [review]
LGTM

**Review by simahawk — APPROVED** [review]
LG

**Comment by jbaudoux** [comment]
Should have been `product_packaging_calculator` in product-attribute :(

**Comment by sebalix** [comment]
Discussed with @jbaudoux :

1. we create this new `product_packaging_calculator` in `OCA/product-attribute`
2. we update `stock_packaging_calculator` to depend on `product_packaging_calculator` (transitional package)

Will take care of that

**Pull request merged**